build.gradle

```
@@ -85,3 +86,11 @@ jacocoTestCoverageVerification {
 
     dependsOn jacocoTestReport
 }
+
+checkstyle  {
+	toolVersion = "10.12.5"
+	configFile = file("${rootDir}/config/checkstyle/checkstyle.xml")
+	configProperties = [
+			"org.checkstyle.google.suppressionfilter.config"     : "suppressions.xml",
+			"org.checkstyle.google.suppressionxpathfilter.config": "suppressions.xml"]
+}
```

이 부분에 필요 없는 공백이 조금 있네요!
아래와 같이 수정하면 좋을 것 같습니다

```suggestion
checkstyle {
    toolVersion = "10.12.5"
    configFile = file("${rootDir}/config/checkstyle/checkstyle.xml")
    configProperties = [
            "org.checkstyle.google.suppressionfilter.config"     : "suppressions.xml",
            "org.checkstyle.google.suppressionxpathfilter.config": "suppressions.xml"]
}
```

---

잡아 주셔서 감사합니다! 뭔가 줄바꿈 작업을 좀 많이 하다보니 컴퓨터가 제대로 잡질 못하는 것 같네요,,, 윈도우에서는 아래 고친 부분 처럼 보여서 인식을 못하고 있었습니다! 감사합니다ㅠㅠㅠ

---

수정이전에는 탭문자가 있었네요! ide 상에서는 스페이스 4개 크기와 동일하게 보이는데 github에서는 크기가 달라서 다르게 보이는 것이었습니다! 스페이스 4개로 수정했습니다~!